### PR TITLE
Correction and redirection

### PR DESCRIPTION
**Pull request by michaelmitnick**
Change of data in the verification request card for Discord.

### CODE REVIEW

File: ui/component/userVerify/view.jsx [hunk 1]
```diff
@@ -120,25 +120,17 @@ class UserVerify extends React.PureComponent<Props> {
...
             subtitle={
               <>
                 <p>
-                  {__(
-                    'A moderator can approve you within our discord server. Please review the instructions within #rewards-approval carefully.'
-                  )}
+                  {__('You can request verification of your Odysee account by sending an email to help@odysee.com')}
                 </p>
-                <p>{__('You will be asked to provide proof of identity.')}</p>
+                <p>{__('Verification requests can take a few hours to be approved.')}</p>
```
Comment (tzarebczan) [inline ui/component/userVerify/view.jsx:130]: @michaelmitnick please add this to the app-strings.json

File: ui/component/userVerify/view.jsx [hunk 2]
```diff
@@ -120,25 +120,17 @@ class UserVerify extends React.PureComponent<Props> {
...
               </>
             }
-            actions={
-              <Button
-                href="https://verification.odysee.com"
-                button="primary"
-                label={__('Join the Odysee Rewards Discord')}
-              />
-            }
+            actions={<Button href="https://chat.odysee.com" button="primary" label={__('Join our Discord')} />}
```
Comment (tzarebczan) [inline ui/component/userVerify/view.jsx:133]: Probably can just remove this button

**Review by michaelmitnick — COMMENTED** [review]
I have made the changes and compiled check. @tzarebczan

File: ui/component/userVerify/view.jsx (hunk 1 above)
Comment (tzarebczan) [reply ui/component/userVerify/view.jsx:131]: meant "'Verification requests can take a few hours to be approved." string to app-strings.json. Let me know if you wanna take another shot.  

Notice you also added back: 'You will be asked to provide proof of identity.' - that's not what I meant, and probably can be removed?

Comment (michaelmitnick) [reply ui/component/userVerify/view.jsx:131]: Oh, that's right, the app-strings.json was not edited. I'll correct it, sorry.

**Comment by tzarebczan** [comment]
thank you, lgtm!

**Pull request merged**